app/components/autocomplete_selector/index.tsx

```diff
@@ -182,7 +182,7 @@ function AutoCompleteSelector({
         if (onSelected) {
             onSelected(selectedOptions);
         }
-    }, [teammateNameDisplay, intl, dataSource]);
+    }, [teammateNameDisplay, intl, dataSource, onSelected]);
```

Comment:
without this and the other line in this file, autocomplete wasn't loading the values until the user navigated to the selector.